.one-pipeline.yaml

```diff
@@ -364,13 +460,6 @@ acceptance-test:
       exit 0
     fi
 
-    SKIP_KIND_E2E_TEST=$(get_env SKIP_KIND_E2E_TEST)
```

Comment:
This option could still be useful if we need to rebuild and scan to quickly validate CVE is resolved, but don't need to run the e2e tests. It's different from the PERIODIC_SCAN, since that one just scans without rebuilding. The cluster spin off in `setup` could be skipped if this option is set.